src/zjs_i2c_handler.c

```diff
@@ -4,7 +4,12 @@
 
 #define I2C_0 0
 #define I2C_1 1
+
+#ifdef CONFIG_BOARD_ARDUINO_101_SSS
+#define MAX_I2C_BUS 1
+#else
 #define MAX_I2C_BUS 2
+#endif
```

Comment:
I am curious, if I2C_1 is a valid bus on Arduino101, eventhough it's not connected to anything, shouldn't we still allow app to open the bus? So instead of fixing the module to limit to only 1 bus, the test case was invalid, since the Arduino 101 and frdm is different